skops/io/tests/test_persist.py

```diff
@@ -807,3 +789,44 @@ def test_get_instance_unknown_type_error_msg():
     msg = "Can't find loader this_get_instance_does_not_exist for type builtins.tuple."
     with pytest.raises(TypeError, match=msg):
         get_instance(state, None)
+
+
+class CustomEstimator(BaseEstimator):
+    """Estimator with np array, np scalar, and sparse matrix attribute"""
+
+    def fit(self, X, y=None):
+        self.numpy_array = np.zeros(3)
+        self.numpy_scalar = np.ones(1)[0]
+        self.sparse_matrix = sparse.csr_matrix(np.arange(3))
+        return self
+
+
+def test_dump_to_and_load_from_disk(tmp_path):
```

Comment:
maybe as an alternative, or another test, we could save/load from disk and memory, and compare the objects' hashes to make sure they're the same?

Reply:
Yes, I can add another test. To be sure I understand correctly, you mean comparing the hashes of both objects in memory, one obtained through `dump/load` and one for `dumps/loads`? If so, what method would you use to hash them, `joblib.hash`?

Reply:
yes, exactly.

Reply:
done